agenta-cli/pyproject.toml

```diff
@@ -28,6 +28,7 @@ python-multipart = "^0.0.6"
 importlib-metadata = "^6.7.0"
 posthog = "^3.1.0"
 pydantic = "1.10.13"
+httpx = "^0.27.0"
```

Comment:
why do we need this ?

Reply:
It's a required dependency when we generate the SDK client backend with fern.